src/TextArea/TextArea.svelte

```diff
@@ -85,7 +85,7 @@
       </label>
       {#if maxCount}
         <div class:bx--label={true} class:bx--label--disabled={disabled}>
-          {value.length}/{maxCount}
+          {value?.length || 0}/{maxCount}
```

Comment:
Let's coalesce the `value` to an empty string.

```suggestion
          {((value ?? '').length)}/{maxCount}
```